Adding IconButton to generate names for wallet names for new_wallet a…

RafiaChy opened the pull request:
1. used suffixIcon to add an IconButton
2. Icon used is refresh.png
3. IconButton tied to generateName function
4.  On clicking IconButton, the wallet name fields of  both new-wallet and restore-wallet gets pasted with generated names

mkyq commented on lib/entities/generate_name.dart:
```diff
@@ -20,5 +20,6 @@ Future<String> generateName() async {
   final chosenNoun = nouns[randomThing.nextInt(nouns.length)];
   final returnString =
       chosenAdjective.capitalized() + ' ' + chosenNoun.capitalized();
+    
```
Why it appeared here ? Do not modify parts which are not effected by your implementation

mkyq commented on lib/src/screens/new_wallet/new_wallet_page.dart:
```diff
@@ -92,44 +98,76 @@ class _WalletNameFormState extends State<WalletNameForm> {
+                      onPressed: () async {
+                        final String rName = await generateName();
+
+                        print(rName);
```
Remove it, as it doesn't require by your implementation

RafiaChy replied:
@mkyq removed print statement

mkyq commented on lib/src/screens/new_wallet/new_wallet_page.dart:
```diff
@@ -92,44 +98,76 @@ class _WalletNameFormState extends State<WalletNameForm> {
+                  decoration: InputDecoration(
+                    suffixIcon: IconButton(
+                      onPressed: () async {
+                        final String rName = await generateName();
```
Do not define type for `final` variables

RafiaChy replied:
@mkyq removed the type

mkyq commented on lib/src/screens/restore/wallet_restore_from_seed_form.dart:
```diff
@@ -57,9 +58,38 @@ class WalletRestoreFromSeedFormState extends State<WalletRestoreFromSeedForm> {
+              margin: const EdgeInsets.only( bottom: 15, left: 13),
+              child: InkWell(
+                onTap: () async {
+                  final String rName = await generateName();
+
+                  print(rName);
```
Same here. Do not define types for `final`s. And remove unnecessary prints.

RafiaChy commented:
@mkyq , I have added a generate-name button for restore-from-key page and 

1. removed empty lines from generate_name.dart, 
2. types for final variables in new_wallet.page.dart and restore_from_seed_form.dart, and 
3. print statements in new_wallet.page.dart and restore_from_seed_form.dart, and
